<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Creation</class>
 <widget class="QDialog" name="Creation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Creer un Compte</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tab_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>1069</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Client </string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QFormLayout" name="client_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::ExpandingFieldsGrow</enum>
         </property>
         <property name="rowWrapPolicy">
          <enum>QFormLayout::WrapAllRows</enum>
         </property>
         <property name="labelAlignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
         <property name="formAlignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="horizontalSpacing">
          <number>2</number>
         </property>
         <property name="verticalSpacing">
          <number>4</number>
         </property>
         <item row="1" column="1">
          <widget class="QLabel" name="genreC">
           <property name="text">
            <string>Genre</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="listGenreC">
           <item>
            <property name="text">
             <string>Mr.</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mme</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="nomC">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="nomLineEditC">
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>400</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignJustify|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="prenomC">
           <property name="text">
            <string>Surname</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="prenomLineEditC">
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>400</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLabel" name="nomDUtilisateurC">
           <property name="text">
            <string>User</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLineEdit" name="nomDUtilisateurLineEditC"/>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="adresseLabelC">
           <property name="text">
            <string>Adresse</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QLineEdit" name="adresseLineEditC">
           <property name="inputMethodHints">
            <set>Qt::ImhNone</set>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="mailC">
           <property name="text">
            <string>Adresse Mail</string>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QLineEdit" name="mailLineEditC">
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>400</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhEmailCharactersOnly</set>
           </property>
          </widget>
         </item>
         <item row="15" column="0">
          <widget class="QLabel" name="telephoneC">
           <property name="text">
            <string>Phone</string>
           </property>
          </widget>
         </item>
         <item row="15" column="1">
          <widget class="QLineEdit" name="telephoneLineEditC">
           <property name="maximumSize">
            <size>
             <width>398</width>
             <height>400</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhDialableCharactersOnly</set>
           </property>
          </widget>
         </item>
         <item row="16" column="0">
          <widget class="QLabel" name="mobileC">
           <property name="text">
            <string>Mobile</string>
           </property>
          </widget>
         </item>
         <item row="16" column="1">
          <widget class="QLineEdit" name="mobileLineEditC">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>400</height>
            </size>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhDialableCharactersOnly</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="enregistrerC">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>26</height>
            </size>
           </property>
           <property name="text">
            <string>Submit</string>
           </property>
           <property name="shortcut">
            <string>Ctrl+S</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="annulerC">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>26</height>
            </size>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
           <property name="shortcut">
            <string>Ctrl+X</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Personnel</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QFormLayout" name="personnel">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::ExpandingFieldsGrow</enum>
         </property>
         <property name="rowWrapPolicy">
          <enum>QFormLayout::WrapAllRows</enum>
         </property>
         <property name="labelAlignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
         <property name="formAlignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
         <property name="horizontalSpacing">
          <number>2</number>
         </property>
         <property name="verticalSpacing">
          <number>4</number>
         </property>
         <item row="0" column="1">
          <widget class="QLabel" name="genre_3">
           <property name="text">
            <string>Gender</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="listGenre">
           <item>
            <property name="text">
             <string>Mr.</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mme</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="titreLabel">
           <property name="text">
            <string>Titel</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="titreComboBox">
           <item>
            <property name="text">
             <string>Docteur</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Professor</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Student</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Azubi</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Secretary</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="nom_2">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="nomLineEdit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>400</height>
            </size>
           </property>
           <property name="cursorMoveStyle">
            <enum>Qt::VisualMoveStyle</enum>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="prenom">
           <property name="text">
            <string>Surname</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="prenomLineEdit">
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>400</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="nomDUtilisateur">
           <property name="text">
            <string>User</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLineEdit" name="nomDUtilisateurLineEdit"/>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="mail">
           <property name="text">
            <string>Adresse Mail</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QLineEdit" name="mailLineEdit">
           <property name="inputMethodHints">
            <set>Qt::ImhEmailCharactersOnly</set>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="adresseLabel">
           <property name="text">
            <string>Adresse</string>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QLineEdit" name="adresseLineEdit">
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>400</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
          </widget>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="telephone_2">
           <property name="text">
            <string>Phone</string>
           </property>
          </widget>
         </item>
         <item row="13" column="1">
          <widget class="QLineEdit" name="telephoneLineEdit">
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>400</height>
            </size>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhDialableCharactersOnly</set>
           </property>
          </widget>
         </item>
         <item row="14" column="0">
          <widget class="QLabel" name="mobile">
           <property name="text">
            <string>Mobile</string>
           </property>
          </widget>
         </item>
         <item row="14" column="1">
          <widget class="QLineEdit" name="mobileLineEdit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>400</height>
            </size>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhDialableCharactersOnly</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="enregistrerP">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>26</height>
            </size>
           </property>
           <property name="text">
            <string>Submit</string>
           </property>
           <property name="shortcut">
            <string>Ctrl+S</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="annulerP">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>26</height>
            </size>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
           <property name="shortcut">
            <string>Ctrl+X</string>
           </property>
           <property name="autoRepeatDelay">
            <number>297</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nomLineEditC</tabstop>
  <tabstop>prenomLineEditC</tabstop>
  <tabstop>nomDUtilisateurLineEditC</tabstop>
  <tabstop>mailLineEditC</tabstop>
  <tabstop>telephoneLineEditC</tabstop>
  <tabstop>mobileLineEditC</tabstop>
  <tabstop>nomLineEdit</tabstop>
  <tabstop>prenomLineEdit</tabstop>
  <tabstop>nomDUtilisateurLineEdit</tabstop>
  <tabstop>mailLineEdit</tabstop>
  <tabstop>adresseLineEdit</tabstop>
  <tabstop>telephoneLineEdit</tabstop>
  <tabstop>mobileLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>annulerP</sender>
   <signal>clicked(bool)</signal>
   <receiver>Creation</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>463</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>annulerC</sender>
   <signal>clicked(bool)</signal>
   <receiver>Creation</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enregistrerC</sender>
   <signal>clicked(bool)</signal>
   <receiver>Creation</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>-3</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enregistrerP</sender>
   <signal>clicked(bool)</signal>
   <receiver>Creation</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>505</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_enregistrerC_clicked()</slot>
 </slots>
</ui>
